docs/en/sql-reference/functions/splitting-merging-functions.md

```diff
@@ -91,20 +91,21 @@ SELECT splitByString('', 'abcde');
 └────────────────────────────┘
 ```
 
-## splitByRegexp(regexp, s)
+## splitByRegexp(regexp, s[, max_tokens])
 
-Splits a string into substrings separated by a regular expression. It uses a regular expression string `regexp` as the separator. If the `regexp` is empty, it will split the string `s` into an array of single characters. If no match is found for this regular expression, the string `s` won't be split.
+Splits a string into substrings separated by a regular expression. It uses a regular expression string `regexp` as the separator. If the `regexp` is empty, it will split the string `s` into an array of single characters. If no match is found for this regular expression, the string `s` won't be split. `max_tokens` limit the number of returned tokens from function `splitByRegexp`.
 
 **Syntax**
 
 ``` sql
-splitByRegexp(regexp, s)
+splitByRegexp(regexp, s[, max_tokens])
```

Comment:
Thank you very much for your advance.  This is exactly the original change I have made, but I reverted it because it breaks  the semantics consistency of function `splitByChar`.   

Now I will add optional argument for all split-family functions.